### ISSUE TITLE
Error: timed out waiting for file

### ISSUE THREAD

**Issue by ocafebabe** [containers/podman#15729]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

I get this error when I try a simple SQL query on my MariaDB container.

**Steps to reproduce the issue:**

It's hard to establish a reproducible scenario in my case, but this is a fresh vanilla install of Ubuntu Server 22.04...

**Describe the results you received:**

`Error: timed out waiting for file /home/mariadb/.local/share/containers/storage/overlay-containers/a0a800ba0881ce593e7c40b2e2e32c40916e9044ec48aa3d152c0403900e2151/userdata/7c2210469927db4fbf6aa14ac1919128b4514f1a9277d7ee0dd340ec9285c1c1/exit/a0a800ba0881ce593e7c40b2e2e32c40916e9044ec48aa3d152c0403900e2151: internal libpod error`

**Additional information you deem important (e.g. issue happens only occasionally):**

I can reproduce it every time, some results are returned after the query execution but it crashes after a few seconds.

**Output of `podman version`:**

```
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.17.3
Built:        Wed Dec 31 19:00:00 1969
OS/Arch:      linux/amd64
```

**Output of `podman info`:**

```
host:
  arch: amd64
  buildahVersion: 1.23.1
  cgroupControllers:
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: 'conmon: /usr/bin/conmon'
    path: /usr/bin/conmon
    version: 'conmon version 2.0.25, commit: unknown'
  cpus: 16
  distribution:
    codename: jammy
    distribution: ubuntu
    version: "22.04"
  eventLogger: journald
  hostname: xo-amtrak-db-ops
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1011
      size: 1
    - container_id: 1
      host_id: 755360
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1010
      size: 1
    - container_id: 1
      host_id: 755360
      size: 65536
  kernel: 5.15.0-1019-azure
  linkmode: dynamic
  logDriver: journald
  memFree: 85841149952
  memTotal: 135081746432
  ociRuntime:
    name: crun
    package: 'crun: /usr/bin/crun'
    path: /usr/bin/crun
    version: |-
      crun version 0.17
      commit: 0e9229ae34caaebcb86f1fde18de3acaf18c6d9a
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/user/1010/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.0.1
      commit: 6a7b16babc95b6a3056b33fb45b74a6f62262dd4
      libslirp: 4.6.1
  swapFree: 17179865088
  swapTotal: 17179865088
  uptime: 19h 39m 4.14s (Approximately 0.79 days)
plugins:
  log:
  - k8s-file
  - none
  - journald
  network:
  - bridge
  - macvlan
  volume:
  - local
registries: {}
store:
  configFile: /home/mariadb/.config/containers/storage.conf
  containerStore:
    number: 2
    paused: 0
    running: 0
    stopped: 2
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/mariadb/.local/share/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 1
  runRoot: /run/user/1010/containers
  volumePath: /home/mariadb/.local/share/containers/storage/volumes
version:
  APIVersion: 3.4.4
  Built: 0
  BuiltTime: Wed Dec 31 19:00:00 1969
  GitCommit: ""
  GoVersion: go1.17.3
  OsArch: linux/amd64
  Version: 3.4.4
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/jammy,now 3.4.4+ds1-1ubuntu1 amd64 [installed]
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/main/troubleshooting.md)**

Yes (podman 3.4.4 is the latest version available on Ubuntu Server 22.04.1 LTS)

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Azure VM - software raid 10 with an XFS filesystem (ftype=1)

**Comment by ocafebabe** [comment]
FYI I get this error only when I launch the query from within the container through a pseudo-tty.

I just did a test with the same query launched externally and it worked perfectly so I guess there's some sort of issue when the output returned by a command is too big for the pseudo-tty. Does it make sense?

**Comment by Luap99** [comment]
Please try with the latest podman version 4.2. I am pretty sure we already fixed this issue (at least I remember the error message) and without a reproducer I cannot confirm.

**Comment by rhatdan** [comment]
Reopen the issues if @Luap99 is mistaken.

**Comment by ocafebabe** [comment]
> Please try with the latest podman version 4.2. I am pretty sure we already fixed this issue (at least I remember the error message) and without a reproducer I cannot confirm.

I would, but like I've already said in the bug's description, podman 3.4.4 is the latest available version on Ubuntu Server 22.04.

**Comment by mheon** [comment]
Upstream is no longer providing support for that version, so there's not much we can do. You may have better luck filing a bug against your distribution.
